rmgpy/pdep/sls.py

```diff
@@ -75,3 +75,25 @@ def get_initial_condition(network,x0,indices):
 
     return p0
 
+def solve_me(M,p0,t):
+    f = Main.eval("""
+function f(u,M,t)
+    return M*u
+end""")
+    jac = Main.eval("""
+function jac(u,M,t)
+    return M
+end""")
+    tspan = (0.0,t)
+    fcn = de.ODEFunction(f,jac=jac)
+    prob = de.ODEProblem(fcn,p0,tspan,M)
+    sol = de.solve(prob,solver=de.CVODE_BDF(),abstol=1e-16,reltol=1e-6)
```

Comment:
Would it be useful to allow abstol and reltol as unser inputs?

Reply:
We could, however, adding a lever like that implies that users should be adjusting it. Right now I think our recommendation is to use the matrix exponential algorithm and the ODE method really only exists (beyond I put it in the paper so it should probably be made available) for users to prove to themselves that the matrix exponential method is working XD. I think the given tolerances should be sufficient for that purpose and if someone is running into sensitive numerics they're going to need to dig around the code anyway.